### PR TITLE
New package: Shemnei.punktf version 3.0.0

### PR DESCRIPTION
**Pull request by michidk**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118141)

Adds https://github.com/Shemnei/punktf by @Shemnei

### CODE REVIEW

File: manifests/s/Shemnei/punktf/3.0.0/Shemnei.punktf.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- RelativeFilePath: punktf-x86_64-pc-windows-msvc/punktf.exe
+  PortableCommandAlias: punktf
+ReleaseDate: 2022-12-23
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Shemnei/punktf/releases/download/v3.0.0/punktf-x86_64-pc-windows-msvc.zip
+  InstallerSha256: 31F4BFBDFD8DD7098CEC2EBD4C8A0F13A6AE21E97CE5F0AECE93919DC99C7E56
+- Architecture: x86
+  InstallerUrl: https://github.com/Shemnei/punktf/releases/download/v3.0.0/punktf-x86_64-pc-windows-msvc.zip
+  InstallerSha256: 31F4BFBDFD8DD7098CEC2EBD4C8A0F13A6AE21E97CE5F0AECE93919DC99C7E56
```
Comment (stephengillie) [inline manifests/s/Shemnei/punktf/3.0.0/Shemnei.punktf.installer.yaml:18]:
```suggestion
  InstallerSha256: 31F4BFBDFD8DD7098CEC2EBD4C8A0F13A6AE21E97CE5F0AECE93919DC99C7E56
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
- Architecture: x86
  InstallerUrl: https://github.com/Shemnei/punktf/releases/download/v3.0.0/punktf-x86_64-pc-windows-msvc.zip
  InstallerSha256: 31F4BFBDFD8DD7098CEC2EBD4C8A0F13A6AE21E97CE5F0AECE93919DC99C7E56
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

Comment (stephengillie) [reply manifests/s/Shemnei/punktf/3.0.0/Shemnei.punktf.installer.yaml:18]: image

Comment (michidk) [reply manifests/s/Shemnei/punktf/3.0.0/Shemnei.punktf.installer.yaml]: Thanks!

Comment (michidk) [reply manifests/s/Shemnei/punktf/3.0.0/Shemnei.punktf.installer.yaml]: Thanks! Are these kind of dependencies yet supported, or would we have to wait for https://github.com/microsoft/winget-cli/issues/163? 
The author just released a static version which does not depend on this runtime. If these kind of dependencies are not yet supported, I would close this PR and create a new one which uses the static dependencies.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**